src/test/java/com/cgdpd/library/service/BookServiceTest.java

```diff
@@ -96,6 +104,86 @@ public void should_throw_exception_if_author_not_exist() {
     }
 
     @Test
+    void should_return_books_with_pagination() {
+        // given
+        var searchCriteria = SearchBookCriteria.builder().build();
+        var spec = byBookSearchCriteria(searchCriteria);
+        var paginationCriteria = PaginationCriteria.builder()
+              .pageIndex(0)
+              .pageSize(10)
+              .sort(Optional.of(new SortParams("title", SortParams.Direction.ASC)))
+              .build();
+        var pageRequest = paginationCriteria.toPageRequest();
+
+        when(bookRepository.findAll(spec, pageRequest))
+              .thenReturn(new PageImpl<>(List.of(
```

Comment:
Create variable with this list, then you can use it for the assertions instead of creating the same exact objects twice